src/sentry/profiles/task.py

```diff
@@ -136,6 +138,21 @@ def process_profile_task(
         set_measurement("profile.stacks.processed", len(profile["profile"]["stacks"]))
         set_measurement("profile.frames.processed", len(profile["profile"]["frames"]))
 
+    if (
+        enabled := options.get("profiling.generic_metrics.functions_ingestion.enabled")
+    ) and project.organization_id in (
+        allowed_orgs := options.get("profiling.generic_metrics.functions_ingestion.allowed_org_ids")
+    ):
+        try:
+            dsn = get_metrics_dsn(project.id)
+            profile["options"] = {
+                "profiling.generic_metrics.functions_ingestion.enabled": enabled,
+                "profiling.generic_metrics.functions_ingestion.allowed_org_ids": allowed_orgs,
```

Comment:
Do you actually need to propagate these two to Vroom? Wouldn't the presence of the `"dsn"` field in `"options"` already imply that the feature is enabled for this profile?